_posts/2019-09-11-how-to-stay-up-to-date.markdown

```diff
@@ -0,0 +1,52 @@
+---
+layout: post
+title: How to stay up to date with your Rails application
+date: 2019-08-27 13:56:00
+categories: ["Rails", "Upgrade"]
+author: bronzdoc
+---
+
+How to stay up to date with your rails application
+
+<!--more-->
+
+In Fastruby.io we work with alot of clients which have outdated Rails applications, we help them upgrade to a newer Rails version.
+
+An outdated Rals application doesn't happen over night, and there are things that you could start doing to avoid going into an outdated project, that will be difficult to maintain and migrate later on.
+
+### Stop monkey patching
+
+In our experience having monkey patched gems when doing an upgrade usually is one of the hardest things to deal with since work hours would have to be put to make the monkey patched gem compatible with the new Rails version. So Have that in mind before dedicating time to monkey patch rails core libraries or gems that depend on an specific rails versions.
+
+### Start treating Deprecations warnings as errors.
+
+In Rails you can configure this option, so everytime a deprecation warning occurs you can address it as soon as you can.
+This will help you to fix the issue rigth a away and be prepared for the next version jump.
+
+```ruby
+  # Raise error on deprecation¬
+  config.active_support.deprecation = :raise
+```
```

Comment:
@bronzdoc Where do you recommend they add this? In all environments or just `test.rb`?  It is not clear to me